File: Sorting/SelectionSort/LeetCode75 Sort Colors/Solution.py
class Solution:
    def sortColors(self, nums):
        """
        :type nums: List[int]
        :rtype: void Do not return anything, modify nums in-place instead.
        """
        for i in range(len(nums)):
            min_val = nums[i]
            min_idx = i
            for j in range(i + 1, len(nums), 1):
                if nums[j] < nums[min_idx]:
                    min_val = nums[j]
                    min_idx = j
            nums[i], nums[min_idx] = nums[min_idx], nums[i]

File: Sorting/SelectionSort/LeetCode75 Sort Colors/Solution.cpp
class Solution {
public:
    void sortColors(vector<int>& nums) {
        for (int i = 0; i < nums.size(); i++){
            int minVal = nums[i];
            int minIndex = i;
            for (int j = i + 1; j < nums.size(); j++){
                if (nums[j] < minVal){
                    minIndex = j;
                    minVal = nums[j];
                }
            }
            swap(nums[i], nums[minIndex]);
        }
    }
};